Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -621,6 +620,12 @@ def import_tests(self):
         for prefixed_value in sorted(total_prefixed_property_values, key=lambda p: total_prefixed_property_values[p]):
             _log.info('  %s: %s', prefixed_value, total_prefixed_property_values[prefixed_value])
 
+        # Fetch the Upstream commit ID for the WPT import So it is ready to insert in the commit message.
```

Comment:
This has a lot of odd capitalisation, and the comment seems entirely redundant given what the next two lines are?

Reply:
crumbles from prototyping